src/utils/normalize-mime-parts.ts

```diff
@@ -135,7 +135,7 @@ export function normalizeMimeParts(
 					);
 				} else if (bodyType && (!acc[bodyType] || disposition !== 'inline')) {
 					if ((bodyType === 'html' || bodyType === 'text') && acc[bodyType]) {
-						acc[bodyType] = acc[bodyType].concat(content);
+						acc[bodyType] = acc[bodyType].concat(content || '');
```

Comment:
can we put default value, the place we are intialising content. In you case, content = part.content || '';